frontend/src/core/resource/components/ResourceMenu.js

```diff
@@ -113,22 +150,70 @@ export class ResourceMenuBase extends React.Component<Props, State> {
                         autoFocus
                         value={ this.state.search }
                         onChange={ this.updateResourceList }
+                        placeholder="Filter resources"
```

Comment:
Please make the placeholder attribute Localizable.

Reply:
<img width="618" alt="Localized error" src="https://user-images.githubusercontent.com/13032234/66938767-19086880-f042-11e9-9edf-1fed0aea2efc.png">

I just realized that my Localized for Resource and Progress doesn't work. I've given an id and add them into the translate.ftl. Did I give wrong id format?

Reply:
Hm, the code looks good. Have you tried running `make build` and then `make run` again?

Reply:
Oh you're right I needed to build and run again.

Reply:
But for the placeholder it doesn't work. How to wrap html attributes? I tried it like this :
`<Localized id="resource-ResourceMenu-filterResources">
                        <input
                            type="search"
                            autoComplete="off"
                            autoFocus
                            value={ this.state.search }
                            onChange={ this.updateResourceList }
                            placeholder="Filter resources"
                        />
                    </Localized>`

Reply:
See https://github.com/mozilla/pontoon/blob/master/frontend/src/modules/machinery/components/Machinery.js#L59.